sql/src/test/java/org/apache/druid/sql/calcite/schema/SystemSchemaTest.java

```diff
@@ -0,0 +1,632 @@
+                              .withConstructor()
+                              .addMockedMethod(
+                                      "handleResponse",
+                                      HttpResponse.class,
+                                      HttpResponseHandler.TrafficCop.class
+                                  )
+                              .addMockedMethod("getStatus")
+                              .createMock();
+    request = EasyMock.createMock(Request.class);
+    authMapper = new AuthorizerMapper(null)
+    {
+      @Override
+      public Authorizer getAuthorizer(String name)
+      {
+        return (authenticationResult, resource, action) -> new Access(true);
+      }
+    };
+
+    final File tmpDir = temporaryFolder.newFolder();
+    final QueryableIndex index1 = IndexBuilder.create()
+                                              .tmpDir(new File(tmpDir, "1"))
+                                              .segmentWriteOutMediumFactory(OffHeapMemorySegmentWriteOutMediumFactory.instance())
+                                              .schema(
+                                                  new IncrementalIndexSchema.Builder()
+                                                      .withMetrics(
+                                                          new CountAggregatorFactory("cnt"),
+                                                          new DoubleSumAggregatorFactory("m1", "m1"),
+                                                          new HyperUniquesAggregatorFactory("unique_dim1", "dim1")
+                                                      )
+                                                      .withRollup(false)
+                                                      .build()
+                                              )
+                                              .rows(ROWS1)
+                                              .buildMMappedIndex();
+
+    final QueryableIndex index2 = IndexBuilder.create()
+                                              .tmpDir(new File(tmpDir, "2"))
+                                              .segmentWriteOutMediumFactory(OffHeapMemorySegmentWriteOutMediumFactory.instance())
+                                              .schema(
+                                                  new IncrementalIndexSchema.Builder()
+                                                      .withMetrics(new LongSumAggregatorFactory("m1", "m1"))
+                                                      .withRollup(false)
+                                                      .build()
+                                              )
+                                              .rows(ROWS2)
+                                              .buildMMappedIndex();
+
+    walker = new SpecificSegmentsQuerySegmentWalker(conglomerate)
+        .add(segment1, index1)
+        .add(segment2, index2)
+        .add(segment2, index2)
```

Comment:
Duplicate line

Reply:
@leventov This is removed [here](https://github.com/apache/incubator-druid/pull/6804/files#diff-1b9272e9d665e89f38ead66e8435bd9dL202) It was because I thought about the replicas in a different way(wrong way), fixed in this new PR.